tests/system/python/conftest.py

```diff
@@ -668,6 +670,29 @@ def _disable_sch(fledge_url, sch_name):
     return _disable_sch
 
 
+@pytest.fixture(scope="function", autouse=True)
+def collect_support_bundle(request, fledge_url):
+    def _collect_support_bundle():
+        PROJECT_ROOT = Path(__file__).absolute().parent.parent.parent.parent
+        
+        try:
+            jdoc = utils.post_request(fledge_url, "/fledge/support", None)
+            assert jdoc["bundle created"]
+            copy_to_cmd = "mkdir -p {0}/support/ && cp -r {1} {0}/support".format(PROJECT_ROOT, jdoc['bundle created'])
```

Comment:
index format - would have negative benefit for me, it only adds noise.
We should use either with {} empty braces or {named} curlies.